homeworks/vitaliy.konak_vitaliykonak/GamesProject/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+        <title>GamesProject</title>
+    </head>
+    <main class="main">
+        <section class="about_section">
+            <h1 class="about_title">About me</h1>
+
+            <div class="about_description">Hi! My name is Vitalii and I'm a Junior Frontend Developer. I am already familiar with main Web Technologies like HTML, CSS, JavaScript and Git version control system.
+            </div>
+
+            <div class="about_link-course">This page was developed during the course
+                <a href="https://www.mastersacademy.education/frontend-for-beginners-it">'Frontend for beginners' from Masters Academy in 2023.</a>
+            </div>
+
+            <div class="about_text">This is a social project from MOCG company where I got an opportunity to work with Frontend mentors and to create my own small project for the portfolio.
+            </div>
+
+            <div class="about_contact">You can contact me  check out my 
+                <a href="https://github.com/vitaliykonak"> GitHub.</a>
+            </div>
+
+        </section>  
+    </main>
+<body>
+    
```

Comment:
`main`  block should be inside the body block. The body block should not be empty

Reply:
@vitaliykonak please fix this